src/test/java/io/ddavison/conductor/FrameworkTest.java

```diff
@@ -26,6 +26,11 @@ public void testSetText() throws Exception {
     }
 
     @Test
+    public void testSetAndValidateText() throws Exception {
+        setAndValidateText("#setTextField", "set and validate text test");
```

Comment:
Yes. I know the option. Just want to give an option so that I have to type something less. In IntelliJ, I have to type a piece of string twice for validation, which is more problematic.